pymc/logprob/utils.py

```diff
@@ -283,3 +286,51 @@ def diracdelta_logprob(op, values, *inputs, **kwargs):
     (const_value,) = inputs
     values, const_value = pt.broadcast_arrays(values, const_value)
     return pt.switch(pt.isclose(values, const_value, rtol=op.rtol, atol=op.atol), 0.0, -np.inf)
+
+
+def find_rvs_in_graph(vars: Union[Variable, Sequence[Variable]]) -> Set[Variable]:
```

Comment:
Shall we add a test for this? And for `find_negated_var` below? There seems to be some warnings from Codecov

Reply:
This seems to have picked changes from main, not this PR.

RE: Codecov, we can't trust. The thing doesn't re-run when you push commits